examples/cp4ba/main.tf

```diff
@@ -18,36 +18,32 @@ resource "null_resource" "mkdir_kubeconfig_dir" {
 
 data "ibm_container_cluster_config" "cluster_config" {
   depends_on        = [null_resource.mkdir_kubeconfig_dir]
-  cluster_name_id   = var.cluster_name_or_id
+  cluster_name_id   = var.cluster_id
   resource_group_id = data.ibm_resource_group.resource_group.id
   config_dir        = local.cluster_config_path
 }
 
-module "cp4ba" {
+module "install_cp4ba" {
   source = "../../modules/cp4ba"
 //  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/cp4ba"
-  enable = true
-
-  # ---- Cluster settings ----
-  cluster_config_path = data.ibm_container_cluster_config.cluster_config.config_file_path
-  ingress_subdomain = var.ingress_subdomain
-
-  # ---- Cloud Pak settings ----
-  cp4ba_project_name      = "cp4ba"
-  entitled_registry_user  = var.entitled_registry_user
-  entitlement_key         = var.entitlement_key
-
+  enable_cp4ba           = var.enable_cp4ba
```

Comment:
This should be forced to `true` since this example is specifically to run the cp4ba module, so it is implied that it is wanted to be enabled.  Don't need to have this as a variable in the examples.

Reply:
Done. It's updated. 
```
  enable_cp4ba           = true
```